docs/src/components/PropTable.js

```diff
@@ -98,7 +98,10 @@ export default function PropTable({
 
   return (
     <Card id="Props" name="Props" showHeading={showHeading}>
-      <Box overflow="auto">
+      <Box
+        overflow="auto"
+        dangerouslySetInlineStyle={{ __style: { overflowY: 'hidden' } }}
```

Comment:
@AlbertCarreras 🤭  - not for this diff but I've been thinking we should add `overflow="autoX"` and `overflow="autoY"` to `Box` so we don't need to do this hack.